packages/gatsby-cli/package.json

```diff
@@ -60,6 +59,9 @@
     "rimraf": "^3.0.2",
     "typescript": "^3.9.5"
   },
+  "peerDependencies": {
+    "react": ">=16.8.0"
+  },
```

Comment:
cli needs react as a dependency to make ink work. If we make it a peerdep, we won't be able to run gatsby-cli out of a gatsby project. React is a peer depedency to gatsby so users have to install it on a project level.